dice_ml/data_interfaces/public_data_interface.py

```diff
@@ -86,21 +71,21 @@ def __init__(self, params):
                 else:
                     self.data_df[feature] = self.data_df[feature].astype(
                         np.int32)
-        self.one_hot_encoded_data = self.one_hot_encode_data(self.data_df)
-        self.encoded_feature_names = [x for x in self.one_hot_encoded_data.columns.tolist(
-            ) if x not in np.array([self.outcome_name])]
-
-        # Initializing a label encoder to obtain label-encoded values for categorical variables
-        self.labelencoder = {}
 
-        self.label_encoded_data = self.data_df.copy()
-
-        for column in self.categorical_feature_names:
-            self.labelencoder[column] = LabelEncoder()
-            self.label_encoded_data[column] = self.labelencoder[column].fit_transform(self.data_df[column])
-
-        #self.train_df, self.test_df = self.split_data(self.data_df)
-        self.train_df = self.data_df
+        # should move the below snippet to gradient based dice interfaces
```

Comment:
since line 76-88 are for gradient-based dice only, moved them to dice-interfaces